plugins/html-plugin/src/main/java/ru/sbtqa/tag/pagefactory/exception/IncorrectElementTypeException.java

```diff
@@ -0,0 +1,29 @@
+package ru.sbtqa.tag.pagefactory.exception;
+
+public class IncorrectElementTypeException extends AssertionError {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this class to remove "Exception" or correct its inheritance. ([squid:S2166](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS2166))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=133&issues=AWfPt2ob4tUZZzFFj6HU&open=AWfPt2ob4tUZZzFFj6HU)

[ISSUE_KEY]: AWfPt2ob4tUZZzFFj6HU